Podfile

```diff
@@ -33,6 +33,12 @@ def tracks
   # pod 'Automattic-Tracks-iOS', :path => '../Automattic-Tracks-iOS'
 end
 
+def wordpress_kit
+  # To allow pod to pick up beta versions use -beta. E.g., 1.1.7-beta.1
+  pod 'WordPressKit', '~> 4.49.0'
+  # pod 'WordPressKit', :git => 'https://github.com/wordpress-mobile/WordPressKit-iOS.git', :branch => ''
```

Comment:
I have opinions on how to define this but I'll leave them for a dedicated PR later on.